source/_integrations/spotify.markdown

```diff
@@ -18,6 +18,7 @@ The `spotify` media player integration allows you to control [Spotify](https://w
 
 - Spotify account
 - Spotify application, properly configured (see below).
+- `internal_url` and `external_url` set correctly. See [Setup basic information](/docs/configuration/basic/)
```

Comment:
. This implies you need to set both, or is a requirement. This is not true.

Reply:
You are right.

Maybe "and/or" would be the best way? In some situation both values have to be set. 

What do you think about it?

Reply:
I think in many cases no values should be set. The root cause lies in something else. If it was configured incorrectly, you should have gotten instruction from Home Assistant, which currently it does not.

I really think we should not add this.

Reply:
Okay, then I will close this pull request.